lighthouse-core/audits/dobetterweb/link-blocking-first-paint.js

```diff
@@ -50,7 +52,10 @@ class LinkBlockingFirstPaintAudit extends Audit {
   static computeAuditResultForTags(artifacts, tagFilter) {
     const artifact = artifacts.TagsBlockingFirstPaint;
 
-    const filtered = artifact.filter(item => item.tag.tagName === tagFilter);
+    const filtered = artifact.filter(item => {
+      return item.tag.tagName === tagFilter &&
+        (item.endTime - item.startTime) * 1000 >= LOAD_THRESHOLD_IN_MS;
```

Comment:
100ms does feel a little long, but that's just my gut. I know we're calling them out in the critical chain, but it would still be nice to catch these faster links here in this audit. Otherwise, users might get confused why it shows up in one place and not the other.